### DISCUSSION TITLE
CentOS 8 BIOS to Alma?

### DISCUSSION THREAD

**Discussion by Thunderstorm919** [AlmaLinux/almalinux-deploy#16] [Q&A]
Is there any plans to allow migration from CentOS 8 BIOS to Alma?  I have tried the script and when it is run, this is what is shown..

[vagrant@centos8-1905-bios-00505696d776 ~]$ sudo bash almalinux-deploy.sh
Check root privileges                                                 OK
Check Secure Boot disabled                                            OK
Check centos-8.x86_64 is supported                                    OK
Download RPM-GPG-KEY-AlmaLinux                                        OK
Import RPM-GPG-KEY-AlmaLinux to RPM DB                                OK
Download almalinux-release package                                    OK
Verify almalinux-release package                                      OK
Remove centos-linux-release package                                   OK
Remove centos-gpg-keys package                                        OK
Remove centos-linux-repos package                                     OK
Verifying...                          ################################# [100%]
Preparing...                          ################################# [100%]
Updating / installing...
   1:almalinux-release-8.3-3.el8      ################################# [100%]
Install almalinux-release package                                     OK
Remove centos-logos package                                           OK
Install almalinux-logos package                                       OK
Last metadata expiration check: 0:00:08 ago on Fri 05 Mar 2021 03:06:25 PM UTC.
Error:
 Problem 1: cannot install the best update candidate for package annobin-9.23-1.el8.x86_64
  - nothing provides librpmio.so.3()(64bit) needed by annobin-9.46-1.el7.x86_64
  - nothing provides librpm.so.3()(64bit) needed by annobin-9.46-1.el7.x86_64
 Problem 2: cannot install the best update candidate for package libidn2-2.2.0-1.el8.x86_64
  - nothing provides libunistring.so.0()(64bit) needed by libidn2-2.3.0-1.el7.x86_64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
Run dnf distro-sync -y. Exit code: 1                                  ERROR
Run dnf distro-sync -y                                                OK

**Comment by Thunderstorm919** [comment]
Dis-reguard, I was able to make it work with a BIOS system

**Reply by ODSG4** [reply]
.
